pkg/collector/corechecks/snmp/internal/profile/profile_resolver.go

```diff
@@ -18,6 +19,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/snmp/internal/configvalidation"
 )
 
+var (
+	profileExpVar = expvar.NewMap("profileErrors")
```

Comment:
I assume this is globally accessible, we may want to give it an NDM/snmp specific name too

Reply:
changed in this commit https://github.com/DataDog/datadog-agent/pull/25233/commits/97230d1ccffb1fff7226f1c4f502e6197165147b